starboard/user.h

```diff
@@ -40,6 +40,7 @@ typedef struct SbUserPrivate SbUserPrivate;
 typedef SbUserPrivate* SbUser;
 
 // A set of string properties that can be queried on a user.
+#if SB_API_VERSION < 16
```

Comment:
Style nit: I'd move the doc comments into the #ifdefd block as well, just to ensure Doxygen doesn't see anything funny later.